server/seeder/index.js

```diff
@@ -0,0 +1,51 @@
+import express from 'express';
+import redis from 'redis';
+import client from '../helpers/redis';
+import * as helpers from './helper';
+import data from './placements';
+
+const router = express.Router();
+
+router.get('/place', (req, res) => {
+  client.set('allocations', JSON.stringify(data), redis.print);
+  res.send(data);
+});
+
+const possiblePlacementStatus = [
+  'Placed - Awaiting Onboarding',
+  'External Engagements - Rolling Off',
+];
+
+const placements = () => new Promise((resolve, reject) => {
+  client.get('allocations', (error, result) => {
+    if (error) {
+      console.log(error);
+      reject(error);
+    }
+    return resolve(result);
+  });
+});
+
+const pipeline = async (req, res) => {
+  let values = await placements();
+  if (!values) {
+    values = await placements();
+    if (!values) throw new Error('no data');
+  }
+  const { query: { status } } = req;
+  const jsonValues = JSON.parse(values);
+  if (!status || !possiblePlacementStatus.includes(status)) {
```

Comment:
What is the purpose of this if statement: `!possiblePlacementStatus.includes(status)`, being that the `status` will be passed by the programmer and is therefore assumed to always be valid, is this validation still necessary?